controllers/users.js

```diff
@@ -0,0 +1,13 @@
+const usersModel = require("../models/users");
+
+// GET api/users
+exports.getUsers = (req, res, next) => {
+  usersModel
```

Comment:
I would say `usersModel` could have an alternative name - for example, in future if you are to interact with users in a different way, how would you name the model?